benchmarks/linear-elastic-plate-with-hole/plot_metrics.py

```diff
@@ -25,12 +24,11 @@ def load_graphs(base_dir):
     return graph_list
 
 
-def query_and_build_table(graph_list):
+def query_and_build_table(graph_list, tools):
```

Comment:
how do we get all triples (from different graphs) in here? I do not think that this works in the current setup. It seems also that our test is not right (the action file only has a kratos).